### PR TITLE
Grammar fix

### PR DESCRIPTION
**Pull request by Shegun-istaken**
I resolved all the errors in spelling and grammar I found in the documents.

### CODE REVIEW

File: All-About-SEO/page-speed.md [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
+with images and videos, the first improvement should be to compress images. There are web applications where this can be done, but since this is a WordPress site, we can
 use one of the many available image compression plugins. An image compression plugin is installed and configured to compress all of the images on the site.  
 
-After images are compressed we can improve performance even more by adding a Content Delivery Network (CDN) to our site. A CDN is a network of proxy servers spanning different
+After images are compressed, we can improve performance even more by adding a Content Delivery Network (CDN) to our site. A CDN is a network of proxy servers spanning different
 geographical locations that present specific site content. This allows images and videos to be cached and readily available for users to access from each of the geographical 
-CDN server locations available. This improves speed for the end user and also improves system performance by reducing bandwidth.
+CDN server locations that are available. This improves speed for the end user and also improves system performance by reducing bandwidth.
```
Comment (ImMattDavison) [inline All-About-SEO/page-speed.md:24]: This line may not be entirely necessary,  `geographical 
CDN server locations available.` does make sense

File: All-About-SEO/page-speed.md [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
-After implementing these changes to our site we find that Pingdom now shows it takes less than 2 seconds to load the home page.    
+After implementing these changes to our site, we find that Pingdom now shows it takes less than 2 seconds to load the home page.    
 
 ### Content Delivery Network
 ![Content Delivery Network](https://i.lensdump.com/i/061Oxz.jpg)   
 
 
 ### Why is it good for SEO?
-Page speed is extremely important for SEO and fast page speeds provide a great user experience. Google has prioritized page speed in the past and in 2020 announced that the upcoming
+Page speed is extremely important for SEO, and fast page speeds provide a great user experience. Google has prioritized page speed in the past and, in 2020, announced that the upcoming
```
Comment (ImMattDavison) [inline All-About-SEO/page-speed.md:33]: First comma appears unnecessary and affects the flow

Comment (PatricNox) [reply All-About-SEO/page-speed.md:33]: These comma addition seems overhead

Comment (ImMattDavison) [reply All-About-SEO/page-speed.md:33]: > These comma addition seems overhead

I would agree here, many of them are necessary but some of them not so much

**Pull request merged**